### DISCUSSION TITLE
Something similar like getStrWidth but for height

### DISCUSSION THREAD

**Discussion by TiMESPLiNTER** [olikraus/u8g2#2145] [Q&A]
I'm looking for the equivalent function to `getStrWidth` but for the height that respects descending characters as well.

Let's say from a font the capital G-ascent from the baseline is 13px and the small g-ascent from the baseline is 8px and g-descent from the baseline is 2px.

* `getStrHeight("gG");` should return 15
* `getStrHeight("g");` should return 10
*  `getStringHeight("G");` should return 13

is there a function to achieve this? I couldn't find it in the library reference.

**Comment by olikraus** [comment]
Sorry for my late reply, but the answer is not that easy. First: Such a function does not exist and indeed you are the first one requesting this. 
I had a look into the code. I think it should be possible to implement such a function, but it will take time to do so. If you still need such a function, then we should create an issue from that discussion.

**Comment by mgagic** [comment]
+1 for this
I found a way around it, but it would've been quite useful to have the feature
